msbuild/Xamarin.Shared/Xamarin.Shared.targets

```diff
@@ -2464,13 +2283,195 @@ Copyright (C) 2018 Microsoft. All rights reserved.
 		</DetectDebugNetworkConfiguration>
 	</Target>
 
+
+	<!--
+
+		Post processing: this is what happens to the app bundle once it's built.
+
+		We must (the order here is important):
+
+		a) Run dsymutil on native binaries
+		b) Strip native binaries
+		c) Notify spotlight about our finished app bundle
+
+		A final step done later is to codesign the app bundle (and all its contained native binaries).
```

Comment:
We notify spotlight before codesign?

Reply:
We've always done it that way, and I don't think it matters, spotlight doesn't look at the code signature afaik.